rllib/examples/serving/dummy_client_with_two_episodes.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
```

Comment:
should we add this as a unit test?

Reply:
Oh yes, I think I forgot this! Let me check. Great catch. It's working though, I checked on my laptop :D

Reply:
done

Reply:
thanks thanks :)